src/script/app.js

```diff
@@ -142,7 +184,13 @@ window.rotatePage = async function (pageID) {
         copyArrayBuffer(filesObject[pageInformation.pdfIndex]),
         pageInformation.pageIndex
     ).then(() => {
-        alert(`Seite ${pageID} wurde erfolgreich gedreht | Page ${pageID} was successfully rotated`);
+        alert(t({
+            de: `Seite ${pageID} wurde erfolgreich gedreht`,
+            en: `Page ${pageID} was successfully rotated`,
+            fr: `La page ${pageID} a été tournée avec succès`,
+            ru: `Страница ${pageID} была успешно повернута`,
+            zh: `页面${pageID}被成功旋转`
+        }))
```

Comment:
Can you add a period or exclamation mark here?

Reply:
done with my last commit